### ISSUE TITLE
Model predict get label name

### ISSUE THREAD

**Issue by Zjq9409** [tensorflow/decision-forests#141]
training data as follows:<br>
class,gender,age,fare,label <br>
1,female,50.0,28.7125,died <br>
1,female,2.0,151.55,died<br>
1,female,25.0,151.55,died<br>
2,female,29.0,26.0,survived<br>
2,female,33.0,26.0,survived<br>
2,female,36.0,26.0,survived<br>
2,female,42.0,26.0,survived<br>

test_data as follows:<br>
class,gender,age,fare <br>
3,male,24,7.1417 <br>
1,male,47,227 <br>

I use ` tfdf.keras.RandomForestModel` predict test data:
```
output = model.predict(test_ds)

[[0.6399997 ]
 [0.13999999]]
```
How can I get specific category names 'died' or 'survived' according to probability?

**Comment by Zjq9409** [comment]
when I use sklearn RandomForestClassifier, the probability of two classifications will be output
```
prediction = model.predict_proba(vec.transform(data))
[[0.27861378 0.72138625]]
[[0.99693835 0.00306163]]
```

**Comment by Zjq9409** [comment]
```
adv_args = tfdf.keras.AdvancedArguments(
    predict_single_probability_for_binary_classification=False
)

model = tfdf.keras.GradientBoostedTreesModel(verbose=0, advanced_arguments=adv_args)
```
I solved this problem by this code.
